subworkflows/nf-core/fasta_newick_epang_gappa/meta.yml

```diff
@@ -0,0 +1,82 @@
+name: "fasta_newick_epang_gappa"
+description: Run phylogenetic placement with a number of query sequences plus a reference alignment and phylogeny. Used in nf-core/phyloplace.
+keywords:
+  - phylogenetic placement
+  - phylogenetics
+  - alignment
+  - fasta
+  - newick
+modules:
+  - hmmer/hmmbuild
+  - hmmer/hmmalign
+  - hmmer/eslalimask
+  - hmmer/eslreformat
+  - epang/place
+  - epang/split
+  - gappa/examinegraft
+  - gappa/examineassign
+  - gappa/examineheattree
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - queryseqfile:
+      type: file
+      description: Fasta file with query sequences
+      pattern: "*.{faa,fna,fa,fasta,fa,faa.gz,fna.gz,fa.gz,fasta.gz,fa.gz}"
+  - refseqfile:
+      type: file
+      description: File with reference sequences; aligned unless an hmmfile is provided
+      pattern: "*.{faa,fna,fa,fasta,fa,phy,aln,alnfaa,alnfna,alnfa,mfa,faa.gz,fna.gz,fa.gz,fasta.gz,fa.gz,phy.gz,aln.gz,alnfaa.gz,alnfna.gz,alnfa.gz,mfa.gz}"
+  - refphylogeny:
+      type: file
+      description: Newick file with reference phylogenetic tree
+      pattern: "*.{newick,tree}"
+  - hmmfile:
```

Comment:
I think it would be great to add what is required. For example `hmmfile` seems optional, right?

Reply:
Good point, done.